localization/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# First, let's create a directory where diagnostics are going to be placed
+make_directory(share/swift/diagnostics)
```

Comment:
Looks like Windows CI doesn't like this because it fails with `ninja: error: 'share/swift/diagnostics', needed by 'localization/CMakeFiles/copy-diagnostics', missing and no known rule to make it`. @compnerd what is the cross-platform way of doing this?

Reply:
Looking at https://github.com/apple/swift/blob/bf47403162fc53281f242da50649fda5b9a193f5/stdlib/cmake/modules/SwiftSource.cmake#L742 and https://cmake.org/cmake/help/latest/command/add_custom_command.html, my suggestion would be to remove this line and instead have `add_custom_command` create the directory -- e.g.

```
...
COMMAND "${CMAKE_COMMAND}" -E make_directory "${output}"
COMMAND "${CMAKE_COMMAND}" -E copy_if_different "${input}/*.yaml" "${output}")
```

Reply:
Thanks, let try that!